source/fuzz/fuzzer_pass_construct_composites.cpp

```diff
@@ -205,10 +185,15 @@ FuzzerPassConstructComposites::FindComponentsToConstructArray(
 
   std::vector<uint32_t> result;
   for (uint32_t index = 0; index < array_length; index++) {
-    result.push_back(available_instructions
-                         ->second[GetFuzzerContext()->RandomIndex(
-                             available_instructions->second)]
-                         ->result_id());
+    if (available_instructions == type_id_to_available_instructions.cend()) {
+      // No suitable instructions are available, so use a zero constant
+      result.push_back(FindOrCreateZeroConstant(
+          element_type_id, GetFuzzerContext()->ChooseEven()));
```

Comment:
Might be good to make all this zeros irrelevant by default (but using PRNG is fine by me too).

Reply:
Yeah, I don't know which is best but OK - I've made them all irrelevant.